cmd/agent/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/latitudesh/agent/internal/client"
```

Comment:
I know that using this approach shortens the code, but just a reminder that we have our [Go SDK](https://github.com/latitudesh/latitudesh-go-sdk)

Reply:
Let's keep it straightfoward in this first version, just to validate it quickly! Then we can revisit and plan it properly